packages/react/src/primitives/Autocomplete/useAutocomplete.tsx

```diff
@@ -60,7 +60,7 @@ export const useAutocomplete = ({
     (option) => option === activeOption
   );
   const activeOptionId =
-    activeOption?.id ||
+    activeOption?.id ??
```

Comment:
You'll see these changes throughout the primitives which fix this error:
https://typescript-eslint.io/rules/prefer-nullish-coalescing/